api/src/main/java/org/pmiops/workbench/auth/ProfileService.java

```diff
@@ -70,7 +75,32 @@ public Profile getProfile(User user) throws ApiException {
     if (user.getAuthorities() != null) {
       profile.setAuthorities(new ArrayList(user.getAuthorities()));
     }
-
+    String userEmailVerificationStatus = null;
+    if (user.getEmailVerificationStatus().equals(EmailVerificationStatus.UNVERIFIED)) {
```

Comment:
Could probably be simplified to if (user.getEmailVerificationStatus().equals(EmailVerificationStatus.PENDING()) {...) else { profile.set...() }